web/packages/teleterm/src/ui/Vnet/vnetContext.tsx

```diff
@@ -149,7 +149,7 @@ export const VnetContextProvider: FC<
     [mainProcessClient]
   );
   const isSupported = platform === 'darwin' || platform === 'win32';
-  const isDiagSupported = platform === 'darwin';
+  const isDiagSupported = platform === 'darwin' || platform === 'win32';
```

Comment:
I think we can remove this variable altogether since all supported platforms now support diag checks. We can always re-add it later if needed.